daemon/logger/journald/read.go

```diff
@@ -245,7 +251,7 @@ drain:
 	return cursor
 }
 
-func (s *journald) followJournal(logWatcher *logger.LogWatcher, config logger.ReadConfig, j *C.sd_journal, pfd [2]C.int, cursor *C.char) *C.char {
+func (s *journald) followJournal(logWatcher *logger.LogWatcher, config logger.ReadConfig, j *C.sd_journal, pfd [2]C.int, cursor *C.char, untilUnixMicro uint64) *C.char {
```

Comment:
Actually it looks like `config` is not used at all in either `followJournal` or `drainJournal`, I think we should take it out.